neurobooth_terra/postgres.py

```diff
@@ -1,4 +1,5 @@
 # Authors: Mainak Jas <mjas@mgh.harvard.edu>
+#        : Siddharth Patel <spatel136@mgh.harvard.edu>
```

Comment:
I think the best practice is to not include authorship in the files themselves. That's what git blame is for.